WRIO.js

```diff
@@ -169,7 +203,32 @@ var CreateTitter = React.createClass({
         <CreateOneTitter data={this.state.data} />
     );	
   }
-
+  /*
```

Comment:
Why it there? Titter markup have to be taken from 
http://wrio.s3-website-us-east-1.amazonaws.com/Titter-WRIO-App/widget/titter.htm